### ISSUE TITLE
After creating alias you can spam the 'Create New Alias' button and TXs are created

### ISSUE THREAD

**Issue by sysmca** [syscoin/blockmarket-desktop-public#108]
After creating an alias the screen should clear all input fields. Currently it's possible to spam the 'Create New Alias' button to resend the data you just sent, and the wallet allows this until the original Alias create TX is confirmed (i think).

I sent the same data 3 times until I got an error saying alias already existed. The alias is listed 3 times in My Aliases. It appears only once in the general Alias search.

**Comment by sysmca** [comment]
Looks like you can do this on any 'create' form (offer, alias, cert)

**Comment by sysmca** [comment]
After I reopened the wallet the alias I 'spam created' 2 times is now listed only once in My Aliases

**Comment by sidhujag** [comment]
In 2.2 creating aliases is a 2 step process with a reservation then activation. With certs and offers it will create new services everytime with different guids.. yea clearing the text on success would probably help here

**Comment by dwasyluk** [comment]
Closing this as duplicate of #91 please continue any discussion there.
